media/js/newsletter/form-utils.es6.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+function checkEmailValidity(email) {
+    return /\S+@\S+/.test(email);
```

Comment:
Do we not want a regex that checks for a TLD? Or does basket handle a more deeper check and this is meant to be simpler?

Reply:
Right, Basket is already well set up to check this so my aim is not to replicate all of the validation logic here. This is just a basic regex check that matches what Firefox uses for it's native in-browser form validation. Most browsers will use the native in-browser check, so this is only really for older browsers.